docs/contracts/v2/guides/08-etherscan.md

```diff
@@ -0,0 +1,289 @@
+---
+draft: false
+id: "etherscan"
+sidebar_position: 8
+title: "Etherscan - Manual Interactions"
+---
+
+## Introduction
+
+Just like with any other truly open protocol, Sablier V2 can be interacted with directly through an explorer like
+Etherscan.
+
+In this guide, we will go over how to create a stream and how to withdraw from a stream by manually interacting with the
+Sablier V2 contracts on Etherscan.
+
+## Creating a Stream
+
+### Prerequisites
+
+Before being able to create a stream using the Sablier V2 Core contracts you need to grant the right allowance. See the
+[Allowances](#allowances) section for a guide on how to do that.
+
+### Step 1: Go to contract page
+
+Head over to our [deployments](/contracts/v2/deployments) list and select the contract address you want to interact
+with. For every chain, that will either be `SablierV2LockupLinear` or `SablierV2LockupDynamic`. In this tutorial, we
+will show how to create a stream using **Linear** variant on Goerli.
+
+Once you found the right contract, click on the address to access its explorer's page. Click on the "Contract" tab, and
+then on the "Write Contract" sub-tab.
+
+![](/img/etherscan-tutorial/01.png)
+
+![](/img/etherscan-tutorial/02.png)
+
+You can now connect your wallet to the interface by clicking on "Connect to Web3".
+
+![](/img/etherscan-tutorial/03.png)
+
+### Step 2: Fill in parameters
+
+We will now proceed by creating our first stream. Let's go with:
+
+- a Linear stream
+- of 20,000 DAI
+- starting Jan 1, 2024 ending Jan 1, 2025
+- with no cliff
+- and non-cancelable
+
+As the start and end date are fixed, we'll be using the `createWithRange` method. Please note, however, that using
+`createWithDurations` is possible too. To learn more about the difference between these two functions, head over
+[here](/contracts/v2/reference/core/interfaces/interface.ISablierV2LockupLinear#createwithdurations).
+
+Open the **"createWithRange"** method, and you start filling in the stream details.
+
+![](/img/etherscan-tutorial/04.png)
+
+```ts
+{
+  sender: "0xe0ae83a6b9cc4f24d0638dc27179f311671e4e2a",
+  recipient: "0xb4bf8a8475d1e8e9a2088f118ad0e2cdc2896183",
+  totalAmount: 20000000000000000000000,
+  asset: "0x97cb342cf2f6ecf48c1285fb8668f5a4237bf862",
+  cancelable: false,
+  range: [1704067200,1704067200,1735689600],
+  broker: ["0x0000000000000000000000000000000000000000",0]
+}
+```
+
+#### Sender
+
+The `sender` is the wallet that will have the ability to cancel the stream, if the stream is cancelable. If it's
+non-cancelable, the sender address will keep some limited capabilities (as mapped
+[here](/contracts/v2/reference/access-control#overview)).
```

Comment:
The sender actually has no capabilities if the stream is non-cancelable. I will adjust this

Reply:
Can't they withdraw on behalf of the sender?

Reply:
oh right, forgot about withdrawals.

that special section sounds overkill; there are a bunch of ways in which we could categorize the state of the contract. I don't find it particularly helpful to zero in on this dimension of cancelable vs non-cancelable (in "Access Control").